packages/side-nav/src/vaadin-side-nav-item.d.ts

```diff
@@ -93,6 +93,24 @@ declare class SideNavItem extends SideNavChildrenMixin(DisabledMixin(ElementMixi
    */
   expanded: boolean;
 
+  /**
+   * Whether to use exact matching when comparing the item's path with the
+   * current browser URL, or not. `true` by default.
+   *
+   * With exact matching, the item is considered current only when the paths
+   * are exactly the same. With non-exact matching, the item is considered
+   * current when the item's path is a prefix of the pathname of the current
+   * browser URL. For example, with exact matching, an item with the path
+   * `/path` is considered current only when the browser URL is `/path`.
+   * With non-exact matching, an item with the path `/path` is considered
+   * current when the browser URL is `/path`, `/path/child`,
+   * `/path/child/grandchild`, etc.
+   *
+   * Note that this only affects matching of the URLs path, not the base
+   * origin or query parameters.
```

Comment:
Let's add `@attr {boolean} match-exact` annotation here and in the corresponding comment in the JS file to make VSCode Lit plugin correctly recognize the dash-case attribute notation.

Reply:
Done.